tests/aeroval/test_setup_classes.py

```diff
@@ -32,34 +31,44 @@ def test_EvalSetup(cfg_exp1: dict):
     assert EvalSetup(**cfg_exp1) == EvalSetup.model_validate(cfg_exp1)
 
 
-@pytest.mark.parametrize(
-    "update,error",
-    [
-        pytest.param(
-            dict(model_cfg=dict(WRONG_MODEL=MODELS["TM5-AP3-CTRL"])),
-            "Invalid name: WRONG_MODEL",
-            id="model_cfg",
-        ),
-        pytest.param(
-            dict(obs_cfg=dict(WRONG_OBS=OBS_GROUNDBASED["AERONET-Sun"])),
-            "Invalid name: WRONG_OBS",
-            id="obs_cfg",
-        ),
-        pytest.param(
-            dict(obs_cfg=dict(OBS=dict(web_interface_name="WRONG_OBS"))),
-            "Invalid name: WRONG_OBS",
-            id="web_interface_name",
-        ),
-    ],
-)
-def test_EvalSetup_INVALID_ENTRY_NAMES(cfg_exp1: dict, error: str):
-    with pytest.raises(EvalEntryNameError) as e:
-        EvalSetup(**cfg_exp1)
-    assert error in str(e.value)
-
-    with pytest.raises(EvalEntryNameError) as e:
-        EvalSetup.model_validate(cfg_exp1)
-    assert error in str(e.value)
+# @pytest.mark.parametrize(
+#     "update,error",
+#     [
+#         pytest.param(
+#             dict(model_cfg=dict(WRONG_MODEL=MODELS["TM5-AP3-CTRL"])),
+#             "Invalid name: WRONG_MODEL",
+#             id="model_cfg",
+#         ),
+#         pytest.param(
+#             dict(obs_cfg=dict(WRONG_OBS=OBS_GROUNDBASED["AERONET-Sun"])),
+#             "Invalid name: WRONG_OBS",
+#             id="obs_cfg",
+#         ),
+#         pytest.param(
+#             dict(
+#                 obs_cfg=dict(
+#                     # obs_vars=("concpm10",),
+#                     OBS=dict(
+#                         obs_vars=("concpm10",),
+#                         web_interface_name="WRONG_OBS",
+#                     ),
+#                 )
+#             ),
+#             "Invalid name: WRONG_OBS",
+#             id="web_interface_name",
+#         ),
+#     ],
+# )
+# def test_EvalSetup_INVALID_ENTRY_NAMES(cfg_exp1: dict, error: str):
+#     # with pytest.raises(EvalEntryNameError) as e:
+#     stp = EvalSetup(**cfg_exp1)
+#     assert stp
+#     # assert error in str(e.value)
+
+
+#     with pytest.raises(EvalEntryNameError) as e:
+#         EvalSetup.model_validate(cfg_exp1)
+#     assert error in str(e.value)
```

Comment:
Remove dead code. 

Is there a need for a similar test for the new implementation?